Adapt PPE to work with recent changes in PyMC

In the meantime pymc has bin pinned <=5.15.1

```python
     def test_ppe_testdata(params):
        csv_data = pd.read_csv(r"testdata/chemical_shifts_theo_exp.csv")
        diff = csv_data.theo - csv_data.exp
        cat_encode = pd.Categorical(csv_data["aa"])
        idx = cat_encode.codes
        coords = {"aa": cat_encode.categories}
        with pm.Model(coords=coords) as model:
            # hyper_priors
            a = pm.Normal("a", mu=0, sigma=10)
            b = pm.HalfNormal("b", 10)
            z = pm.HalfNormal("z", sigma=10)
    
            x = pm.Normal("x", mu=a, sigma=b, dims="aa")  # No prior for this!
    
            y = pm.Normal("y", mu=x[idx], sigma=z, observed=diff)
>       prior, new_prior, pymc_string = pz.ppe(model, params["target"])

preliz/tests/test_ppe.py:109: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
preliz/predictive/ppe.py:57: in ppe
    bounds, prior, p_model, var_info, var_info2, draws, free_rvs = get_model_information(model)
preliz/ppls/pymc_io.py:134: in get_model_information
    idxs = [free_rvs.index(var_) for var_ in nc_parents]
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

.0 = <list_iterator object at 0x7f5f97998880>

>   idxs = [free_rvs.index(var_) for var_ in nc_parents]
E   ValueError: ExpandDims{axis=0}.0 is not in list
```

--- a/preliz/ppls/pymc_io.py
+++ b/preliz/ppls/pymc_io.py
@@ -204,8 +204,8 @@ def reshape_params(model, var_info, p_model, params):
 def non_constant_parents(var_):
     """Find the parents of a variable that are not constant."""
     parents = []
-    for variable in var_.get_parents()[0].inputs[3:]:
+    for variable in var_.get_parents()[0].inputs[2:]:
         if not isinstance(variable, TensorConstant):
-            parents.append(variable)
+            parents.append(variable.owner.inputs[0])
 
     return parents

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -9,5 +9,5 @@ nbconvert===6.5.4
 nbformat==5.1.3
 lxml==5.0.0
 ipykernel
-pymc<=5.15.1
+pymc>=5.16.0
 bambi>=0.13.0

